### DISCUSSION TITLE
CANT ACCESS XSERVER AFTER DNS FIX

### DISCUSSION THREAD

**Discussion by alonssro** [microsoft/WSL#6555] [General]
# Environment

```none
Windows build number:  10.0.19042.804
Your Distribution version: 20.04
Whether the issue is on WSL 2 and/or WSL 1: WSL 2
```

# Steps to reproduce

After fixing WSL 2 DSN problem: 

Firewall allows public and private trffic from xserver.

![resolve.conf](https://user-images.githubusercontent.com/13211107/107840384-f2faf180-6d77-11eb-9d1e-9ba4963d9504.png)

![wsl.conf](https://user-images.githubusercontent.com/13211107/107840437-49683000-6d78-11eb-9185-db8d1c73c31b.png)

This fixed my DNS issue but after this I can't access XSERVER apps. Just doesn't work. 

#  Expected behavior

Run xservers with the following settings:

![image](https://user-images.githubusercontent.com/13211107/107840476-b2e83e80-6d78-11eb-83ba-b20200edc131.png)
![image](https://user-images.githubusercontent.com/13211107/107840483-b976b600-6d78-11eb-8482-f04d8f56b150.png)
![image](https://user-images.githubusercontent.com/13211107/107840488-c09dc400-6d78-11eb-8e88-f38b75afdae9.png)
 
run `xeyes` or `google-chrome` should display the apps. 

# Actual behavior

This is the only thing that shows on the terminal

![image](https://user-images.githubusercontent.com/13211107/107840508-e3c87380-6d78-11eb-94fb-12e9afc35ab3.png)

**Comment by therealkenc** [comment]
Your X server is not on 8.8.8.8. You can get the IP address of Windows with:
```
$ sudo apt install iproute2
$ ip route show default
```

Looks like:

![image](https://user-images.githubusercontent.com/7049700/107979805-0f637d80-6f74-11eb-9268-43a51d262157.png)

**Reply by alonssro** [reply]
@therealkenc  If I put my IP Address on resolv.conf, I cannot access internet neither  xserver.

**Reply by therealkenc** [reply]
Your screencap indicates your `DISPLAY` variable is set to `8.8.8.8`. `8.8.8.8` is not the address of your xserver.

**Reply by alonssro** [reply]
@therealkenc I put the ip address from the command

```
$ sudo apt install iproute2
$ ip route show default
```

And still didn't work

**Reply by therealkenc** [reply]
Similar exercise as #5959 to debug the problem. Something is blocking your connection. Firewall sometimes. Or something is otherwise misconfigured. Or whatever externality motivated you to "DNS FIX" is causing problems, possibly.

Run the following and post a screencap the output of:
```
$ export DISPLAY=$(ip route show default | awk '{print $3}'):0
$ echo $DISPLAY
$ netstat.exe /an | grep 6000
$ xeyes
```

On good day, looks like:

![image](https://user-images.githubusercontent.com/7049700/108431238-72b11200-71f7-11eb-8720-952180a28eb9.png)

**Reply by alonssro** [reply]
@therealkenc 
![image](https://user-images.githubusercontent.com/13211107/108447147-fc2b0900-7224-11eb-91b7-c0a785c90fc7.png)

This is the screeshot. Not working
I'm running also XServer

**Reply by alonssro** [reply]
@therealkenc 

I return back all the configuration from my 'DNS Fix' motivation, and still it's not working

![image](https://user-images.githubusercontent.com/13211107/108450453-db65b200-722a-11eb-8b8c-1e616fe8525f.png)
![image](https://user-images.githubusercontent.com/13211107/108450741-4ca56500-722b-11eb-91ba-6ec3168ebd50.png)
